eth_vertigo/test_runner/truffle/truffle_runner.py

```diff
@@ -13,7 +13,7 @@
 
 def _make_temp_truffle_directory(original_dir: str):
     td = mkdtemp()
-    copy_tree(original_dir, td)
+    copy_tree(original_dir, td, preserve_symlinks=1)
```

Comment:
```suggestion
    copy_tree(original_dir, td, preserve_symlinks=True)
```
Why did you choose to use an int vs a bool here?

Reply:
@JoranHonig Because the Python 2 official docs used the `int` 1 for it while I tried to google for Python 3 docs for this. However, ended up without result. Have you tested using `True` can also work? If so, it's obviously be better to use `True`.

Reply:
I see, after reviewing the documentation I could find onlne on distutils' copy_tree it does indeed expect to get an integer. I'll resolve this comment